src/Perl6/bootstrap.c/BOOTSTRAP.nqp

```diff
@@ -3666,6 +3666,7 @@ BEGIN {
 	#     has str $!longname;
```

Comment:
Just noticed the pre-existing-but-errant tab here, but the next line could use a `# has Lock $!lock;` also.